numpy/core/src/umath/loops.c.src

```diff
@@ -571,6 +571,22 @@ PyUFunc_O_O_method(char **args, npy_intp *dimensions, npy_intp *steps, void *fun
         PyObject **out = (PyObject **)op1;
         PyObject *ret = PyObject_CallMethod(in1 ? in1 : Py_None, meth, NULL);
         if (ret == NULL) {
+            PyObject *exc, *val, *tb;
+            PyErr_Fetch(&exc, &val, &tb);
+            if (in1 == NULL) {
+                /* When can this ever happen? */
```

Comment:
This branch is never hit, not clear to me why it is needed, but it reflects the `in1 ? in1 : Py_None` from above

Reply:
I think `NULL` is intended as a just-in-case alias for None, so the message doesn't need to distinguish them.